packages/cli/generators/project/templates/package.json.ejs

```diff
@@ -1,11 +1,14 @@
 {
   "name": "<%= project.name -%>",
-  "version": "1.0.0",
+  "version": "0.0.1",
```

Comment:
What's wrong with `1.0.0` in the scaffolded project? This change seems rather unrelated to yarn support to me 🤷🏻

Reply:
Let's keep `0.0.1` as is.

Reply:
Before this time that I was missing I had already spoken with @raymondfeng to set the version to 0.0.1. I should still separate it into a different commit.